File: Software Technologies/Exam Preparation III - LogNoziroh & ShoppingList/ShoppingList/Java Solution/src/main/java/shoppinglist/entity/Product.java
package shoppinglist.entity;

import javax.persistence.*;

@Entity
@Table(name = "products")
public class Product {
	//TODO: Implement me ...

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer id;

    @Column
    private Integer priority;

    @Column
    private String name;

    @Column
    private Integer quantity;

    @Column
    private String status;

    public Product() {
    }

    public Product(Integer priority, String name, Integer quantity, String status) {
        this.priority = priority;
        this.name = name;
        this.quantity = quantity;
        this.status = status;
    }

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public Integer getPriority() {
        return priority;
    }

    public void setPriority(Integer priority) {
        this.priority = priority;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public Integer getQuantity() {
        return quantity;
    }

    public void setQuantity(Integer quantity) {
        this.quantity = quantity;
    }

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }
}

File: Software Technologies/Exam Preparation III - LogNoziroh & ShoppingList/ShoppingList/JavaScript Solution/models/product.js
const Sequelize = require('sequelize');

module.exports = function(sequelize){

    let Product = sequelize.define('Product', {
        priority:{type: Sequelize.INTEGER, required: true, allowNull:false,},
        name:{type: Sequelize.TEXT, required: true, allowNUll:false, },
        quantity:{type: Sequelize.INTEGER, required: true, allowNull:false,},
        status:{type: Sequelize.TEXT, required: true, allowNull:false,}
    }, {
        timestamps:false
    });
    return Product;
};